yii/image/drivers/Kohana/Image/GD.php

```diff
@@ -657,6 +660,14 @@ protected function _save_function($extension, & $quality)
                                 // Use a compression level of 9 (does not affect quality!)
                                 $quality = 9;
                         break;
+                        case 'webp':
+                                // Save a Webp file
+                                $save = 'imagewebp';
+                                $type = IMAGETYPE_WEBP;
+
+                                // Quality is different in webp than PNG
+                                $quality = 85;
```

Comment:
Thanks for the PR. It's better not to set `$quality` value here ([`webp`](https://www.php.net/manual/en/function.imagewebp.php) handling should be similar to the one for [`jpeg`](https://www.php.net/manual/en/function.imagejpeg.php), *not* [`gif`](https://www.php.net/manual/en/function.imagegif.php) or [`png`](https://www.php.net/manual/en/function.imagepng.php)).

With the PR-suggested change, end-users won't have control over the resulting `webp` images quality, even [passing `$quality`](https://github.com/yurkinx/yii2-image#usage) to `save`/`render` functions. Note that `$quality` is passed by reference to [_save_function](https://github.com/yurkinx/yii2-image/blob/590795d0e94c909c636cba1007a619012e118ac9/yii/image/drivers/Kohana/Image/GD.php#L628). So we get the value of 85 to be "hard-coded", instead of the expected behavior.